source/cpu/power_amd64.go

```diff
@@ -74,3 +74,57 @@ func discoverSSTBF() (bool, error) {
 
 	return false, nil
 }
+
+func discoverSSTCP() (bool, error) {
+	/*
+	/ There are two requirements for SST-CP being enabled on a machine:
+	/ 1) Turbo is enabled
+	/ 2) There is at least two different values for EPP (Energy Performance Preference) among the cores of the system
+	*/
+
+	epp_file := "cpufreq/energy_performance_preference"
+	epp_values := []string{}
+
+	// 1) Check Turbo is enabled
+	features, _ := detectPstate()
+	if features["turbo"] == "false" {
+		return false, nil
+	}
```

Comment:
I started to think that maybe we could/should cache the features somewhere, e.g. in the `Source` struct itself. But maybe that's out of the scope of this patch and should be done separately.